app/controllers/sessions_controller.rb

```diff
@@ -1,23 +1,25 @@
 class SessionsController < ApplicationController
   
-  def new; end
+  def new
+    redirect_to home_path if logged_in?
+  end
 
   def create
-    user = User.find_by(email: params[:email])
+    user = User.where(email: params[:email]).first
```

Comment:
Why did you change it from `.find_by` ? `where(...).first` is inefficient for single records.